<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckUpWindowClass</class>
 <widget class="QMainWindow" name="CheckUpWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheckUp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>450</width>
      <height>300</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <widget class="QWidget" name="tab_geooverlaps">
     <attribute name="title">
      <string>Geometry</string>
     </attribute>
     <widget class="QListWidget" name="overlaplist">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>380</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Overlaps:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOverlaps">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color : green; }</string>
      </property>
      <property name="text">
       <string>No Overlaps have been found</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_optics">
     <attribute name="title">
      <string>Optics</string>
     </attribute>
     <widget class="QLabel" name="labelOptics">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color : green; }</string>
      </property>
      <property name="text">
       <string>Simulation is not Wavelength Resolved</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="opticsTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_pmts">
     <attribute name="title">
      <string>PMs</string>
     </attribute>
     <widget class="QTableWidget" name="pmtsTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_particles">
     <attribute name="title">
      <string>Interactions</string>
     </attribute>
     <widget class="QListWidget" name="listInteraction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelInteraction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Serif</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color : green; }</string>
      </property>
      <property name="text">
       <string>No Interaction data seems to be missing</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonRefreshAll">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Refresh All</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRefreshCurrent">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
